mustache-templates/updateLocation.mst

```diff
@@ -110,19 +110,19 @@
                 <div class="form-group row justify-content-start row-no-gutters">
                     <label for="initialTimer" class="col-sm-3 col-form-label">Initial Timer (ms):</label>
                     <div class="col-sm-5">
-                        <input type="text" class="form-control" name="initialTimer" placeholder="Initial Timer (milliseconds)" value="{{initialTimer}}" required>
+                        <input type="text" class="form-control" name="initialTimer" placeholder="Initial Timer (seconds)" value="{{initialTimer}}" required>
                     </div>
                 </div>
                 <div class="form-group row justify-content-start row-no-gutters">
                     <label for="reminderTimer" class="col-sm-3 col-form-label">Reminder Timer (ms):</label>
                     <div class="col-sm-5">
-                        <input type="text" class="form-control" name="reminderTimer" placeholder="Reminder Timer (milliseconds)" value="{{reminderTimer}}" required>
+                        <input type="text" class="form-control" name="reminderTimer" placeholder="Reminder Timer (seconds)" value="{{reminderTimer}}" required>
                     </div>
                 </div>
                 <div class="form-group row justify-content-start row-no-gutters">
                     <label for="fallbackTimer" class="col-sm-3 col-form-label">Fallback Message Timer (ms):</label>
                     <div class="col-sm-5">
-                        <input type="text" class="form-control" name="fallbackTimer" placeholder="Fallback Message Timer (milliseconds)" value="{{fallbackTimer}}" required>
+                        <input type="text" class="form-control" name="fallbackTimer" placeholder="Fallback Message Timer (seconds)" value="{{fallbackTimer}}" required>
```

Comment:
I think these two are still in ms. Unless you want to change them now too?

Reply:
ah, yeah you're right. Switching them back, I think changing them is for another task.